ui/lib/pki/addon/components/page/pki-tidy-status.hbs

```diff
@@ -0,0 +1,106 @@
+<Toolbar>
+  <ToolbarActions>
+    {{! TODO remove action here in favor of cancel action in status banner }}
+    <button
+      type="button"
+      class="toolbar-link"
+      {{on "click" (fn (mut this.confirmCancelTidy) true)}}
+      data-test-confirm-cancel-tidy
+    >
+      Cancel tidy
+      <Icon @name="chevron-right" />
+    </button>
+    <div class="toolbar-separator"></div>
+    <button
+      type="button"
+      class="toolbar-link"
+      {{on "click" (fn (mut this.tidyOptionsModal) true)}}
+      data-test-pki-tidy-options-modal
+    >
+      Tidy
+      <Icon @name="chevron-right" />
+    </button>
+    {{#if @autoTidyConfig.enabled}}
+      <ToolbarLink @route="tidy.auto" data-test-pki-auto-tidy-config>
+        Auto-tidy configuration
+      </ToolbarLink>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+
+{{! TIDY OPTIONS MODAL }}
+<Modal
+  @type="info"
+  @title="Tidy this mount"
+  @onClose={{fn (mut this.tidyOptionsModal) false}}
+  @isActive={{this.tidyOptionsModal}}
+  @showCloseButton={{true}}
+>
+  <section aria-label="tidy-options-modal-content" class="modal-card-body">
+    <h3 class="title is-5">Tidy operation</h3>
+    <p class="has-text-grey has-bottom-padding-s">
+      Tidying cleans up the storage backend and/or CRL by removing certificates that have expired and are past a certain
```

Comment:
When we're closer to completion we should get feedback from Alex on this description. I think he had some ideas on how to elaborate here